### PR TITLE
fix(jemalloc): More gracefully handle cmd failure due to missing types and symbols

### PR DESCRIPTION
**Pull request by fidgetingbits**
Similar to #2519 fixing jemalloc_heap test by catching exceptions, this adds a bit more robust handling of expected failure cases in the other jemalloc commands, like if required symbols or types aren't found, rather than just throwing exceptions.

### CODE REVIEW

File: pwndbg/commands/heap.py [hunk 1]
```diff
@@ -1641,17 +1650,15 @@ def jemalloc_heap() -> None:
...
-
-    extents = rtree.extents
-    if len(extents) == 0:
-        print(message.warn("No extents found"))
+        extents = rtree.extents
+        if len(extents) == 0:
+            print(message.warn("No extents found"))
+            return
+        for extent in extents:
+            # TODO: refactor so not create copies
```
Comment (disconnect3d) [inline pwndbg/commands/heap.py:1658]: wdym by copies here?

**Review by disconnect3d — CHANGES_REQUESTED** [review]
pls change the `def jemalloc_find_extent(addr) -> None:` to `def jemalloc_find_extent(addr) -> bool:` to fix lint (mypy) issues

File: pwndbg/commands/heap.py (hunk 1 above)
Comment (fidgetingbits) [reply pwndbg/commands/heap.py:1658]: fwiw this is me refactoring code to add try except only. Original code (including that comment) is by @jetchirag and I haven't reviewed that part closely yet, so not sure.

**Comment by fidgetingbits** [comment]
> pls change the `def jemalloc_find_extent(addr) -> None:` to `def jemalloc_find_extent(addr) -> bool:` to fix lint (mypy) issues

not part of my changes, but will do. fwiw likely worth changing lint.sh to run mypy outside of ci to more easily catch these during dev?

**Comment by fidgetingbits** [comment]
Fixed

**Review by disconnect3d — APPROVED** [review]

**Pull request merged**